### PR TITLE
added new ecoset models; fixes #36

### LINKED ISSUE #36: add VGG weights for ecoset
add a custom model for VGG without batch norm

### PR DESCRIPTION
**Pull request by hahahannes**

### CODE REVIEW

**Comment by hahahannes** [comment]
This will add new alexnet, vgg16, resnet50 and inception models trained on ecoset and a normalized loss function. Also added custom models to tests to increase covergage
